src/datastore/database.hpp

```diff
@@ -18,16 +18,23 @@
 class Database
 {
 public:
+    /* Aliases */
+    using completedEdge = std::vector<std::tuple<int,int,int>>;  //"from" <-> "to" and distance
```

Comment:
1 complete edge = a vector? doesn't make sense

Reply:
I'll make the tuple a completed edge